### PR TITLE
AEROGEAR-3414 Add certificate check and notification

### PR DESCRIPTION
**Pull request by aidenkeating**
Show a notification when a certificate check fails with a link out to a documentation page.

### CODE REVIEW

**Comment by aidenkeating** [comment]
@wei-lee @StephenCoady Would you mind taking a look?

At the moment it's catching SSL errors and returning a generic message. We could add a details section with the original error message.

screen shot 2018-07-10 at 10 33 50

@ciaranRoche I know you worked on the Cordova version of this, mind taking a quick look too?

**Review by ciaranRoche — COMMENTED** [review]
Looks good, I can't run locally, but the code follows the same pattern as cordova and android :+1:

File: ios-showcase-template.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -990,17 +964,17 @@
...
+				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = "ios-showcase-template/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = "org.aerogear.ios-showcase-template";
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE = "cff102ff-c079-4ed6-a8cb-27f8ec14cbd8";
-				PROVISIONING_PROFILE_SPECIFIER = "match InHouse org.aerogear.ios-showcase-template";
+				PROVISIONING_PROFILE = "";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```
Comment (wei-lee) [inline ios-showcase-template.xcodeproj/project.pbxproj]: can we revert some of the changes around the code signing in this file? The settings are required to get it build on circleci.

**Review by StephenCoady — APPROVED** [review]
Tested it using different invalid certs from https://badssl.com/ and it works perfectly.

**Review by wei-lee — COMMENTED** [review]
looks good. just one comment

File: ios-showcase-template.xcodeproj/project.pbxproj (hunk 1 above)
Comment (aidenkeating) [reply ios-showcase-template.xcodeproj/project.pbxproj]: Oh good catch! Will do

**Comment by aidenkeating** [comment]
@wei-lee Updated

**Review by wei-lee — APPROVED** [review]

**Pull request merged**